cogs/moderation/moderation.py

```diff
@@ -124,15 +124,14 @@ async def _ban(self, ctx: Context, member: discord.Member, *, reason="Breaking T
 
             await ctx.send(embed=embed)
 
-
     @commands.command(aliases=['tban'])
     @commands.guild_only()
     @commands.has_permissions(ban_members=True)
-    async def tempban(self, ctx: Context, member: discord.Member, length : CustomTimeConverter, *, reason="Breaking The Rules"):
+    async def tempban(self, ctx: Context, member: discord.Member, length: CustomTimeConverter, *,
```

Comment:
This is regarding line 138, but...I don't understand github and I can't comment there.
> `expires = curtime + timedelta(seconds=length)`
So the CustomTimeConverter already gives you a `datetime.timedelta` object after parsing the arg, so you can directly add that with `curtime`, like
```py
expires = curtime + length
```